.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   build:
     env:
       ATHENS_MONGO_STORAGE_URL: mongodb://localhost:27017
-      ATHENS_MINIO_ENDPOINT: localhost:9000
+      ATHENS_MINIO_ENDPOINT: http://localhost:9000
```

Comment:
This won't be breaking for existing Minio users because we trim the protocol later in the config, however, would this be breaking for existing AWS users who do not leverage a protocol in their configuration?

@nrwiersma what do you think? The context is that the newest AWS SDK requires a protocol.

Reply:
On second review, I think this is fine.